vassal-app/src/main/java/VASSAL/build/module/Chatter.java

```diff
@@ -554,7 +554,7 @@ public void keyCommand(KeyStroke e) {
 
     if ((e.getKeyCode() == 0 || e.getKeyCode() == KeyEvent.CHAR_UNDEFINED)
       && !Character.isISOControl(e.getKeyChar())) {
-      if (((e.getModifiers() & KeyEvent.ALT_DOWN_MASK) != 0) || (e.getModifiers() & KeyEvent.CTRL_DOWN_MASK) != 0)  {
+      if ((e.getModifiers() & KeyEvent.ALT_DOWN_MASK) != 0 || (e.getModifiers() & KeyEvent.CTRL_DOWN_MASK) != 0 || (e.getModifiers() & KeyEvent.ALT_GRAPH_DOWN_MASK) != 0)  {
```

Comment:
Would it not be faster to do a single compare of e.getModifiers() by or-ing the 3 masks together?  
`if ((e.getModifiers() & (KeyEvent.ALT_DOWN_MASK | KeyEvent.CTRL_DOWN_MASK | KeyEvent.ALT_GRAPH_DOWN_MASK)) != 0)  {`

Reply:
Not necessarily faster (if Alt key down), but definitely cleaner, which is more important. Thanks! Fixed.